server/middlewares/index.js

```diff
@@ -0,0 +1,47 @@
+import jwt from 'jsonwebtoken';
+import db from '../models';
+
+const { User } = db;
+/**
+ *Users endpoint middlewares
+ *
+ * @export
+ * @class UsersMiddleware
+ */
+export default class UsersMiddleware {
+  /**
+   *Validates token
+   * @returns {undefined}
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   * @memberof UsersMiddleware
+   */
+  static async verifyToken(req, res, next) {
```

Comment:
Probably this could be made to serve better if its renamed (`validUser` maybe) and made to load the valid bearer (user) onto the request object as `req.user`?